test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionServiceTests.cs

```diff
@@ -297,11 +297,11 @@ public async Task ConnectToDefaultDatabaseRespondsWithActualDbName([Values("mast
                     Connection = connectionDetails
                 });
 
-            Assert.Multiple(() =>
-            {
-                Assert.That(connectionResult.ConnectionId, Is.Not.Empty, "ConnectionId");
-                Assert.NotNull(connectionResult.ConnectionSummary, "ConnectionSummary");
-                Assert.AreEqual(expectedDbName, connectionResult.ConnectionSummary.DatabaseName, "I expect connection to succeed and the Summary to include the correct DB name");
+            Assert.Multiple(() =>
```

Comment:
looks like some whitespace updates made by the editor...the new test case is at the end of file.